server/container/guice/guice-common/src/main/java/org/apache/james/utils/DataProbeImpl.java

```diff
@@ -58,13 +58,18 @@ public void addUser(String userName, String password) throws Exception {
         usersRepository.addUser(userName, password);
     }
 
+    public DataProbeImpl fluentAddUser(String userName, String password) throws Exception {
```

Comment:
why do you need to create a new method for that ? changing addUser to return `this` should be enough ?

Reply:
The "big deal" is JMX silly stuff that expects serialization, thus we can not touch the DataProbe API

Reply:
Ok, I see, could we add a  ̀.fluent() ̀ method that return a fluent API instead ?